packages/gateway/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gateway-nft-storage",
+  "version": "0.0.0",
+  "description": "IPFS gateway for nft.storage",
+  "private": true,
+  "type": "module",
+  "module": "./dist/index.mjs",
+  "scripts": {
+    "build": "node scripts/cli.js build",
+    "dev": "miniflare --watch --debug",
+    "deploy": "wrangler publish --env production",
+    "pretest": "npm run build ",
+    "test": "npm-run-all -p -r mock:ipfs.io test:worker",
+    "test:worker": "ava --verbose test/*.spec.js",
+    "mock:ipfs.io": "smoke -p 9081 test/mocks/ipfs.io"
```

Comment:
Can we spin up an IPFS node instead? We can use the IPFS HTTP API to add content that we want to fetch.

Reply:
Let's do this in the follow up PR (Race), I just commented to remember: https://github.com/nftstorage/nft.storage/pull/961#pullrequestreview-845775128

That's because I need to have tests where I can manipulate who is faster. In that PR it will be easier to do that, perhaps using a real ipfs node + smoke tests for other gateways

Reply:
Yes fine to do in follow up PR 😄 

> That's because I need to have tests where I can manipulate who is faster. In that PR it will be easier to do that, perhaps using a real ipfs node + smoke tests for other gateways

Maybe just a simple proxy server that adds a delay? I find smoke to have quite a big overhead in terms of knowing what to call the file, what the response object should look like, how to get requests parsed correctly etc.